client/flags/flags.go

```diff
@@ -141,6 +142,7 @@ func RegisterRestServerFlags(cmd *cobra.Command) *cobra.Command {
 	cmd.Flags().Uint(FlagMaxOpenConnections, 1000, "The number of maximum open connections")
 	cmd.Flags().Uint(FlagRPCReadTimeout, 10, "The RPC read timeout (in seconds)")
 	cmd.Flags().Uint(FlagRPCWriteTimeout, 10, "The RPC write timeout (in seconds)")
+	cmd.Flags().Bool(FlagUnsafeCORS, false, "The RPC cors are disabled by default")
```

Comment:
Let's come up with a better flag description than:

> "The RPC cors are disabled by default"

Maybe something along the lines of:

> Allows CORS requests from all domains. For development purposes only, use it at your own risk.